pam/integration-tests/vhs-helpers_test.go

```diff
@@ -95,6 +95,14 @@ var (
 	vhsSleepRegex = regexp.MustCompile(
 		`(?m)\$\{?(AUTHD_SLEEP_[A-Z_]+)\}?(\s?([*/]+)\s?([\d.]+))?(.*)$`)
 	vhsEmptyLinesRegex = regexp.MustCompile(`(?m)((^\n^\n)+(^\n)?|^\n)(^─+$)`)
+
+	// vhsWaitSuffix adds support for Wait+Suffix /Pattern/ command.
+	vhsWaitSuffix = regexp.MustCompile(`\bWait\+Suffix(@\S+)?[\t ]+(/(.*)/|(.*))`)
+	// vhsWaitPromptRegex adds support for Wait+Prompt /Pattern/ command.
+	vhsWaitPromptRegex = regexp.MustCompile(`\bWait\+Prompt(@\S+)?[\t ]+(/(.*)/|(.*))`)
+	// vhsWaitNth adds support for Wait+Nth(X) /Pattern/ command, where X is the
+	// number of values of the same content we want to match.
+	vhsWaitNth = regexp.MustCompile(`\bWait\+Nth\((\d+)\)(@\S+)?[\t ]+(/(.*)/|(.*))`)
```

Comment:
Please explain what those different commands do, it's not clear to me from the comments

Reply:
Ok. I've added more verbose comments now.